solvers/stereo_imaging/time_window_pruned_stereo_milp/src/milp_model.py

```
@@ -0,0 +1,737 @@
+    timeout_reached = False
+    solve_time = 0.0
+
+    solve_start = time.perf_counter()
+
+    if backend in ("auto", "ortools"):
+        try:
+            selected_indices, objective_value, solve_stats = solve_with_ortools(model, time_limit_s)
+            backend_used = "ortools"
+            solve_time = solve_stats.get("solve_time_s", 0.0)
+            timeout_reached = solve_stats.get("status") != "OPTIMAL"
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)